loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/Networks.java

```diff
@@ -0,0 +1,272 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.loadflow.validation;
+
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.stream.StreamSupport;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Iterators;
+
+import eu.itesla_project.commons.io.table.Column;
+import eu.itesla_project.commons.io.table.CsvTableFormatterFactory;
+import eu.itesla_project.commons.io.table.TableFormatter;
+import eu.itesla_project.commons.io.table.TableFormatterConfig;
+import eu.itesla_project.iidm.network.Bus;
+import eu.itesla_project.iidm.network.Line;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.TwoWindingsTransformer;
+
+/**
+ *
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class Networks {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Networks.class);
+
+    private Networks() {
+    }
+
+    private static double EPSILON_X = 0.01;
+    private static TableFormatterConfig TABLE_FORMATTER_CONFIG = TableFormatterConfig.load();
+    private static Column[] COLUMNS = {
+        new Column("id"),
+        new Column("characteristic"),
+        new Column("value")
+    };
+    
+    private static Writer createStdOutputWriter() {
+        return new OutputStreamWriter(System.out) {
+            @Override
+            public void close() throws IOException {
+                flush();
+            };
+         };
+    }
+    
+    public static boolean checkFlows(String id, double r, double x, double rho1, double rho2, double u1, double u2, double theta1, double theta2, double alpha1,
```

Comment:
This method was public in the original Networks class. I kept it, just changed for providing the config as input